src/libraries/Microsoft.PowerFx.Core/Syntax/Nodes/VariadicOpNode.cs

```diff
@@ -43,7 +43,7 @@ internal override TexlNode Clone(ref int idNext, Span ts)
                 newNodes.Add(Children[i], children[i]);
             }
 
-            return new VariadicOpNode(ref idNext, Op, children, Clone(OpTokens, ts), SourceList.Clone(ts, newNodes));
+            return new VariadicOpNode(ref idNext, Op, children, Clone(OpTokens.ToArray(), ts), SourceList.Clone(ts, newNodes));
```

Comment:
>Clone [](http://example.com/codeflow?start=64&length=5)

Can we have an overload of Clone that takes IReadOnlyList<T> ?   so no source change on this line. 

.ToArray() creates a copy, and then Clone() will create a 2nd copy.   A new overload can also ensure just one copy.

Reply:
added overload